Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project=".\scripts\sdk\Sdk.props" />
 
   <PropertyGroup>
-    <LangVersion>8</LangVersion>
+    <LangVersion>9</LangVersion>
```

Comment:
We need to update the readme as well - C# 9 requires at least VS 16.7, though for .NET 5.0 it has to be 16.8.

Reply:
Setting version to 16.8+ in wiki and README 
(for simplicity. 16.7 is a servicing baseline though, otherwise just upgrading in itself is the problem.)